sacc/tracer.py

```diff
@@ -36,8 +73,23 @@ def meanZ(self):
         if self.z is None :
             return -1
         return (self.z*self.Nz).sum()/self.Nz.sum()
-            
+
+    def is_CL(self):
+        """Is the tracer a cluster stack. Returns a boolean."""
+        return (self.Mproxy_name is not None) and (self.type == "spin0")
+
+    def is_WL(self):
+        """Is the tracer a source for a lensing measurement. Returns a boolean.
+        TODO: is the check for the Mproxy_name necessary? All spin2 things could be a WL tracer...
```

Comment:
It is true that all spin2 things could be a WL tracer. I think we need to tweak the tracer class for a user to be able to specify this is a WL tracer, not like other spin2 things such as CMB polarizations. We need to discuss with @slosar how to do it.